src/parser.cpp

```diff
@@ -89,6 +96,7 @@ void XASMParser::instruction() {
                         operandDest();
                         match(TokenType::Comma);
                         operandSrc();
+                        pc += 2;
```

Comment:
Please move `pc += 2` at the end of the function as the program counter is incremented in every case of the switch

Reply:
Done